README.md

```diff
@@ -299,6 +315,38 @@ The value for `SPEED` must be a valid positive integer.
 
 ---
 
+### test
+
+Verifies if the MediaRemote Adapter is functioning correctly.
+
+This can be integrated into your app to help confirm that our adapter is still effective and select fallback options if necessary since future macOS updates may break it again.
+
+**Usage**
+
+```
+/usr/bin/perl /path/to/mediaremote-adapter.pl /path/to/MediaRemoteAdapter.framework /path/to/NowPlayingTestClient test
+```
+
+Note that the `test` command requires the absolute path to the `NowPlayingTestClient` helper executable as the second argument (before the command name).
+
+**Output**
+
+The command outputs a single character to stdout:
+- `0` - MediaRemote Adapter is working correctly
+- `1` - MediaRemote Adapter is not working correctly
+
+**How it works**
+
+1. First attempts to get now playing information normally
+2. If no media is detected, launches a helper process (`NowPlayingTestClient`) that simulates media playback
+3. Attempts to retrieve now playing information again
+4. Terminates the helper process
+5. Reports whether the adapter can successfully detect the simulated media
+
+**IMPORTANT: May interfere with other apps using MediaRemote**: The test can create a fake media entry that will briefly appear as the now playing app. This only happens when no other media is playing. Since the helper process has no bundle identifier, it is mostly ignored by the `stream` and `get` commands — `stream` won't update, and `get` will print `null`. However, both commands will also print error messages to stderr about the missing bundle identifier.
```

Comment:
> However, both commands will also print error messages to stderr about the missing bundle identifier.

Just remove this, I will fix that because the error makes no sense:

```objc
        printErr(
            [NSString stringWithFormat:@"Failed to determine bundle identifier "
                                       @"for process with PID %d",
                                       pid]);
```

That is executed when this condition meets:

```objc
    NSRunningApplication *process =
        [NSRunningApplication runningApplicationWithProcessIdentifier:pid];
    if (process == nil) {
```

Feel free to fix it in this PR, it should just return false.